js/src/components/paid-ads/campaign-assets-form.test.js

```diff
@@ -205,7 +205,10 @@ describe( 'CampaignAssetsForm', () => {
 		render(
 			<CampaignAssetsForm
 				validate={ alwaysValid }
-				initialCampaign={ { amount: 10, level: 'high' } }
+				initialCampaign={ {
+					amount: 10,
+				} }
+				currentAmount={ 12 }
```

Comment:
@ankitrox The `currentAmount` should theoretically have the same value as `amount`.